pulsar-broker/src/main/java/com/yahoo/pulsar/broker/namespace/OwnershipCache.java

```diff
@@ -130,9 +127,16 @@
                             if (LOG.isDebugEnabled()) {
                                 LOG.debug("Successfully acquired zk lock on {}", namespaceBundleZNode);
                             }
+                            final OwnedBundle bundle = new OwnedBundle(
+                                    ServiceUnitZkUtils.suBundleFromPath(namespaceBundleZNode, bundleFactory));
+                            // update the version
+                            localZkCache.getZooKeeper().exists(path, false, (rc1, path1, ctx1, stat) -> {
```

Comment:
yes, I think I mixed with [disconnect-retry](https://github.com/yahoo/pulsar/blob/master/pulsar-zookeeper-utils/src/main/java/com/yahoo/pulsar/zookeeper/ZooKeeperSessionWatcher.java#L146) with session-expired. if session is expired then broker anyway won't be able to delete zknode as there is no zk-session.
So, I think as there is no update on this znode then we don't need this patch.